### DISCUSSION TITLE
Get coordinates for image partition using 'ocr_only'

### DISCUSSION THREAD

**Discussion by yixinzhang12** [Unstructured-IO/unstructured#880] [Q&A]
Hello everyone, 

I have been trying to get coordinates when partitioning images (png). I was able to get coordinates if I set strategy='hi_res'. But I would get "None" if I switch strategy to be 'auto' or 'ocr_only'. 

Is there a way to get the relative coordinates for images with strategy='ocr_only'?

Any help would be appreciated. 

Thank you!

**Comment by yixinzhang12** [comment]
I went ahead and just used pytessearct.image_to_data() for my pipeline. It'll still be nice to hear if there is a built in way of extracting coordinates using unstructure.io. Thanks!

**Reply by MthwRobinson** [reply]
Thanks @yixinzhang12 ! Added #983 to address this in the library.
